install.sh

```diff
@@ -70,5 +70,13 @@ else
 
 fi
 
+# Install Haskell
+export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
+curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+cabal install --lib HUnit
+cabal install --lib gloss
+cabal install HLint
```

Comment:
```suggestion
cabal install hlint
```

![image](https://user-images.githubusercontent.com/19409687/191051495-1dea2151-f71a-4db8-bb20-948017e60288.png)